DiscordBot/Commands/EventCommands.cs

```diff
@@ -8,11 +8,12 @@
 using DSharpPlus.CommandsNext;
 using DSharpPlus.CommandsNext.Attributes;
 using DSharpPlus.Entities;
-using DSharpPlus.Interactivity;
 using DSharpPlus.Interactivity.Extensions;
 
 namespace DiscordBot.Commands
 {
+    [Group("event")]
+    [RequireRoles(RoleCheckMode.All, "Bot Whisperer")]
```

Comment:
What's this for? 🙂

Reply:
So the [Group] attribute makes it so that every command in this class can be invoked with ?event [commandName].
The RequireRoles attribute checks if the user that invoked the command has all of the speicified roles (meaning people without it can't use the commands)

Reply:
Ah okay. And is "Bot Whisperer" the role you want to go with? Has that been configured/do you have a plan for configuring it so that only the relevant people have that role? 🙂

Reply:
Currently that's the name I've picked and is what I'm using in the test server. This can easily be changed after asking the softwire discord moderators which they prefer. The plan is to have the moderators just assign the role to the dedicated event managers to give them access to the commands

Reply:
Okay, sounds fine. Probably worth documenting this somewhere though, for example in the readme? 🙂

Reply:
Will do :D